manage-data/data-store/data-streams/reindex-tsds.md

```diff
@@ -9,208 +9,111 @@ products:
   - id: elasticsearch
 ---
 
-# Reindex a TSDS [tsds-reindex]
+# Reindex a time series data stream [tsds-reindex]
```

Comment:
@kkrik-es if I understand correctly, this reindexing manual is suggesting to reindex the data of one data stream into a single backing index of another data stream. Right?

If this is true, then I think we need to add a disclaimer before a user gets unpleasantly surprised. 

We could also mention the reindex data stream API that was added for upgrades, I will check if it works.